src/api/remote.rs

```diff
@@ -30,6 +32,11 @@ use crate::{
     },
 };
 
+/// Used to filter for main branch in an integration
+pub const MAIN_BRANCH: &str = "main";
+/// Used to filter for master branch in an integration
+pub const MASTER_BRANCH: &str = "master";
```

Comment:
These, and the general concept of "branches" and "tags", belong in the `git` module; generic remotes do not have the concept of branches or tags.